Raven.Database/Plugins/Builtins/HotSpareReplicationBehavior.cs

```diff
@@ -279,26 +281,30 @@ public void CheckHotSpareLicenseStats(object state = null)
 
         }
         
-        private void ReportLicensingUsage(string id, ReportHotSpareUssage.ActivationMode mode )
+        private void ReportLicensingUsage(string id, ReportHotSpareUsage.ActivationMode mode)
         {
-            try
+            Thread.Sleep(10000000);
```

Comment:
I was testing the hot spare activation and didn't want to really activate it.
It was removed in the second commit :)